indico/modules/events/timetable/templates/display/indico/_contribution.html

```diff
@@ -5,15 +5,19 @@
                                                               render_notes, render_description, render_time %}
 
 {% macro render_contribution(contrib, event, parent=none, nested=false, hide_end_time=false, timezone=none,
-                             show_notes=false) -%}
-    <li class="timetable-item timetable-contrib">
+                             show_notes=false, show_location=false) -%}
+     {% set anchor -%}
```

Comment:
I'd probably go for this:

    {% set anchor = 'c-%d-%s' | format(contrib.friendly_id, contrib.title|slugifx(maxlen=30)) %}